tests/functional/includes/fn.bind.polyfill.js

```diff
@@ -0,0 +1,51 @@
+// Needed for PhantomJS < 2.0
+
+if (!Function.prototype.bind) {
```

Comment:
If I were to use the npm package, we would end up with polyfills all over the place (some returned by the polyfill service, some under `node_modules/`, and some that are not available as npm packages would stay in the `includes/` directory) This would quickly become a mess. Also, note that this polyfill is temporary since Phantom 2 (soon available) will not require it.